.licenseignore

```diff
@@ -0,0 +1,2 @@
+vendor
+pkg/descheduler
```

Comment:
The descheduler reuses some code from upstream. We should keep the Kubernetes license to follow the Apache license.